src/components/data/data-utils.js

```diff
@@ -64,10 +65,11 @@ export const getDownloadCommand = (fileName, gsUri, accessUrl) => {
 }
 
 export const getUserProjectForWorkspace = async workspace => (workspace && await canUseWorkspaceProject(workspace)) ?
-  workspace.workspace.googleProject :
+  isGoogleWorkspace(workspace) :
```

Comment:
This looks wrong. `getUserProjectForWorkspace` should return a string (the Google project), not a boolean.

Reply:
Ah good catch -- mean to remove this